src/character.cpp

```diff
@@ -2230,6 +2237,9 @@ hp_part Character::body_window( const std::string &menu_header,
             if( bleed > 0.0f ) {
                 desc << arrow << tag_colored_string( string_format( _( "Chance to stop: %d %%" ),
                                                      int( bleed * 100 ) ), c_light_green ) << "\n";
+            } else {
+                desc << arrow << tag_colored_string( _( "This item will not stop the bleeding." ),
```

Comment:
No, because examining wounds call this with no healing bonuses. But if you use a bandage for example on NPC it will show you the standard shape of the menu.